tests/ssl-opt.sh

```diff
@@ -1686,6 +1686,23 @@ CLI_OUT="cli_out.$$"
 PXY_OUT="pxy_out.$$"
 SESSION="session.$$"
 
+# If MBEDTLS_ENTROPY_C && MBEDTLS_ENTROPY_NV_SEED && !MBEDTLS_NO_PLATFORM_ENTROPY
```

Comment:
Do we really need that sophistication? `all.sh` just unconditionally creates a 64 bytes `tests/seedfile` and it seems to work well enough. Creating a seedfile even if we don't need it is not a problem, is it?

Reply:
It is not relative CI tests. For my local development, I usually call above command. I think developer might meet same issue and waste time to debug it.